### PR TITLE
Fix SENSEI & Ascent component in AMReX.cmake

### PR DESCRIPTION
**Pull request by c-wetterer-nelson**
This fixes a little bug in the AMReX.cmake file where the SENSEI component was misnamed. 

The correct component name is documented here:
https://amrex-codes.github.io/amrex/docs_html/BuildingAMReX.html#importing-amrex-into-your-cmake-project

### CODE REVIEW

**Comment by c-wetterer-nelson** [comment]
@ax3l, I totally missed this little typo. Hopefully won't take much to merge this fix in.

File: cmake/dependencies/AMReX.cmake [hunk 1]
```diff
@@ -219,7 +219,7 @@ macro(find_amrex)
             set(COMPONENT_PIC)
         endif()
         if(WarpX_SENSEI)
-            set(COMPONENT_SENSEI AMReX_SENSEI)
+            set(COMPONENT_SENSEI SENSEI)
```
Comment (ax3l) [inline cmake/dependencies/AMReX.cmake:222]: Correct, had this as part of  #2209

I'll push the fix for line 202 as well :)

Comment (ax3l) [reply cmake/dependencies/AMReX.cmake:222]: Done :)

**Review by ax3l — APPROVED** [review]
Thanks a lot! :)

**Comment by c-wetterer-nelson** [comment]
Ah, I wasn't even thinking about Ascent. Thanks for adding that!

**Comment by ax3l** [comment]
Yeah, I was looking at one of them in #2209 and realized "wait a second..." :D

Great that we got this updated, because when we get the `find_package` logic sorted out I can stop for 21.09+ to vendor AMReX in Spack before some catches me doing that `*whistles nervously*`

**Pull request merged**